kivy/core/camera/camera_avfoundation.pyx

```diff
@@ -66,6 +68,15 @@ class CameraAVFoundation(CameraBase):
         avf_camera_get_image(storage.camera,
             &width, &height, &rowsize, &data)
 
+        cyarr = cyview.array(
+            shape=((rowsize * height),),
```

Comment:
just saw that now, maybe it's a remainder of debugging too :)
```suggestion
            shape=(rowsize * height,),
```

Reply:
🥶 Fixed and rebased. Thank you